src/main/docs/guide/cache/cacheConfiguration.adoc

```diff
@@ -0,0 +1,25 @@
+To use the MicroStream cache abstraction, you must declare a dependency on
+
+dependency:micronaut-microstream-cache[group="io.micronaut.microstream"]
+
+To use the Micronaut cache annotations, you need to declare a dependency on the Micronaut cache module:
```

Comment:
Maybe is better to have an `api` dependency on cache-core to avoid this?

Reply:
done https://github.com/micronaut-projects/micronaut-microstream/pull/23/commits/3676bc8755d989cbbef9543b2121241f61a3e4d2